cfbs.json

```diff
@@ -1153,6 +1153,99 @@
         "json cfbs/def.json def.json"
       ]
     },
+    "stig-red-hat-enterprise-linux-7": {
+      "description": "Red Hat Enterprise Linux 7 Security Technical Implementation Guide",
+      "tags": ["security", "experimental"],
+      "repo": "https://github.com/nickanderson/cfengine-security-hardening",
+      "by": "https://github.com/nickanderson",
+      "version": "0.0.1",
+      "commit": "adf4fd65eeddf2007c4cd0592955d39323dc191e",
+      "subdirectory": "stig/red_hat_enterprise_linux_7/",
+      "dependencies": ["compliance-report-imports"],
+      "steps": [
+        "copy ./compliance-reports/compliance-report.json ./.no-distrib/compliance-report-definitions/red-hat-enterprise-linux-7-security-technical-implementation-guide.json",
+        "copy ./config.cf services/cfbs/modules/stig-red-hat-enterprise-linux-7/config.cf",
+        "copy ./V-204392/V-204392.cf services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204392.cf",
+        "copy ./V-204424/V-204424.cf services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204424.cf",
+        "copy ./V-204425/V-204425.cf services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204425.cf",
```

Comment:
Why all the individual `copy`'s?

Reply:
Why not?

Reply:
Looks tedious and error-prone to maintain